packages/content/rollup.config.js

```diff
@@ -1,3 +1,6 @@
 const rollupUtils = require('../../util/rollup-utils');
+const commonJS = require('rollup-plugin-commonjs');
 
-module.exports = rollupUtils.config('content');
+const config = rollupUtils.config('content')
```

Comment:
Since we ended up not doing the modification we did this change for, just roll it back to how it was before

```js
module.exports = rollupUtils.config('content');
```